dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -8257,6 +8264,8 @@ public static final class Name {
     public static final String FUSE_DEBUG_ENABLED = "alluxio.fuse.debug.enabled";
     public static final String FUSE_FS_NAME = "alluxio.fuse.fs.name";
     public static final String FUSE_LOGGING_THRESHOLD = "alluxio.fuse.logging.threshold";
+    public static final String FUSE_MAX_READER_CONCURRENCY =
+        "alluxio.max.fuse.reader.concurrency";
```

Comment:
typo? I think the convention is to keep the string literal property name the same as the constant's name.

```suggestion
    public static final String FUSE_MAX_READER_CONCURRENCY =
        "alluxio.fuse.max.reader.concurrency";
```